### ISSUE TITLE
Import RNASeqV2 raw_counts and scaled-estimate

### ISSUE THREAD

**Issue by AmyOlex** [zhandong/TCGA2STAT#2]
Would it be possible to add in support for obtaining the RSEM "raw_counts" and "scaled_estimate" values for the RNASeq2 function?  The authors of RSEM recommend using the rounded "raw_counts" for input into programs like DESeq2 and EdgeR for differential analysis, and the "scaled_estimates" can be converted to TPM-like values according to the RSEM authors.  Being able to import these values with your excellent tool would be useful.  The current RNASeq2 values that are imported are the normalized RSEM counts, which, from my understanding, are not appropriate for input into programs like DESeq2 because they do their own normalization.  Thank You!

**Comment by turgeonmaxime** [comment]
@AmyOlex I think what you're asking for is not tier 3 data. In other words, you may need special authorization for downloading the data. But I may be wrong about that...

**Comment by AmyOlex** [comment]
@turgeonmaxime Yes it is Tier 3.  You can find it in the ...RSEM_genes file that is not normalized.  At this point the TCGA has been migrated over to the Genomic Data Commons.  Will you be editing your tool to support that?
